utils.js

```diff
@@ -0,0 +1,39 @@
+const { NETWORKS } = require('./constants');
+
+const numberToHexString = (number) => number.toString(16);
+
+const isValidNetwork = (network) => {
+    if (!NETWORKS[network]) {
+        throw new Error(`Network ${network} is not valid value (valid values are: ${Object.keys(NETWORKS)})`);
+    }
+    return true;
+};
+
+const signedNumberToHexStringLE = (number) => {
+
+    let numberAsHex = number.toString(16);
```

Comment:
```suggestion
    let numberAsHex = numberToHexString(number);
```
Since we already have that nice function over there, we can make use of it